scale/scale/settings.py

```diff
@@ -189,33 +192,54 @@ def get_env_boolean(variable_name, default=False):
 AUTHENTICATION_BACKENDS = [
     'django.contrib.auth.backends.ModelBackend',
 ]
-
-REST_FRAMEWORK = {
-    'DEFAULT_AUTHENTICATION_CLASSES': (
-        'rest_framework.authentication.SessionAuthentication',
-        'rest_framework.authentication.TokenAuthentication',
-
-        ###############
-        # Social Auth #
-        ###############
-        'oauth2_provider.contrib.rest_framework.OAuth2Authentication',
-        'rest_framework_social_oauth2.authentication.SocialAuthentication',
-    ),
-    'DEFAULT_PERMISSION_CLASSES':
-        ('util.rest.ScaleAPIPermissions',),
-    'DEFAULT_FILTER_BACKENDS': (
-        'django_filters.rest_framework.DjangoFilterBackend',
-    ),
-    'DEFAULT_PAGINATION_CLASS': 'util.rest.DefaultPagination',
-    'DEFAULT_RENDERER_CLASSES': (
-        'rest_framework.renderers.JSONRenderer',
-        'rest_framework.renderers.BrowsableAPIRenderer',
-        'rest_framework.renderers.AdminRenderer',
-    ),
-    'ALLOWED_VERSIONS': ('v6', 'v7'),
-    'DEFAULT_VERSION': 'v6',
-    'DEFAULT_VERSIONING_CLASS': 'rest_framework.versioning.NamespaceVersioning',
-}
+if AUTHENTICATION_ENABLED:
```

Comment:
Can we change this so the entire block isn't duplicated? For example when `AUTHENTICATION_ENABLED` is true, just do:

```
REST_FRAMEWORK['DEFAULT_AUTHENTICATION_CLASSES'] = (...)
REST_FRAMEWORK['DEFAULT_PERMISSION_CLASSES'] = ('util.rest.ScaleAPIPermissions',)
```